make/protogen.mk

```diff
@@ -27,6 +27,7 @@ GENERATED_API_SRCS = $(ALL_PROTOS_REL:%.proto=$(GENERATED_BASE_PATH)/%_grpc.pb.g
 GENERATED_API_GW_SRCS = $(SERVICE_PROTOS_REL:%.proto=$(GENERATED_BASE_PATH)/%.pb.gw.go)
 GENERATED_API_SWAGGER_SPECS = $(API_SERVICE_PROTOS:%.proto=$(GENERATED_BASE_PATH)/%.swagger.json)
 GENERATED_API_SWAGGER_SPECS_V2 = $(API_SERVICE_PROTOS_V2:%.proto=$(GENERATED_BASE_PATH)/%.swagger.json)
+CUSTOM_SWAGGER_SPECS = $(shell find central/docs/api_custom_routes -name "*.swagger.json")
```

Comment:
nitpick: It would be good to define `central/docs/api_custom_routes` as a make variable (similar to `PROTO_BASE_PATH`). In that case, we can use that variable here and also when we call `mergeswag.sh`.